integration-tests/src/test/kotlin/org/stellar/anchor/platform/test/Sep24End2EndTests.kt

```diff
@@ -118,7 +118,7 @@ class Sep24End2EndTest(
     // Submit transfer transaction
     val txn = (anchor.getTransaction(withdrawal.id, token) as WithdrawalTransaction)
     val transfer =
-      wallet.stellar().transaction(txn.from).transferWithdrawalTransaction(txn, asset).build()
+      wallet.stellar().transaction(keypair).transferWithdrawalTransaction(txn, asset).build()
```

Comment:
Is this intentional?

Reply:
Yes, it is necessary due to a wallet version upgrade